src/components/toolbar/button.js

```diff
@@ -18,20 +18,22 @@ const ToolbarButton = ({
   ...otherProps
 }) => {
   const elem = (
-    <Button
-      title={title}
-      onClick={onClick}
-      disabled={disabled}
-      className={clsx('button', type && `button-${type}`, active && 'button--active')}
-      {...otherProps}
-    >
-      {icon && <Icon path={icon} size={'26px'} className={'villain-icon'} />}
-      {label && <span className={'villain-label'}>{label}</span>}
-      {children && <span className={'villain-label'}>{children}</span>}
-    </Button>
+    <span tabIndex={0} className={clsx('button-wrapper')}>
```

Comment:
Will this affect the styling / layout of the toolbar and buttons ?

Reply:
The only difference is that the tooltip shows up while hovering a larger area around the button than before, because the tooltip is now watching the span, which is larger than the button because of the margin and padding on the button itself.

I can move those styles to the span, it will just take some more css

Reply:
ok, that will be better, thanks :v: